dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/FATSuite.java

```diff
@@ -66,7 +66,8 @@
                 SlowAppStartTest.class,
                 JsonbTest.class,
                 JsonpTest.class,
-                ManagedBeansTest.class
+                ManagedBeansTest.class,
```

Comment:
Please remember to update the copyright date in the header of any file which has functionally changed (any change except whitespace or formatting).

Reply:
Done, thanks!